packages/curve-ui-kit/src/shared/ui/TooltipContent.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { InvertTheme } from './ThemeProvider'
+
+const { Spacing } = SizesAndSpaces
+
+// note: use this in the Tooltip component to remove default (inverted color) padding
+export const OmitTooltipPadding = { tooltip: { sx: { '&': { padding: 0 } } } }
+
+export const TooltipContent = ({ title, children }: { title: ReactNode; children: ReactNode }) => (
```

Comment:
Maybe make the `title` property optional? I assume we're going to need to use this for every tooltip we have due to the theme inversion, and sometimes tooltips don't have a title.

```suggestion
export const TooltipContent = ({ title, children }: { title?: ReactNode; children: ReactNode }) => (
```

Reply:
For shorter tooltips I assumed the title would be the only thing

Reply:
We can always change if there is a use case